### ISSUE TITLE
HTTP endpoints do not return full JSON response

### ISSUE THREAD

**Issue by fyrbach** [WrenSecurity/wrends#78]
## Summary
HTTP endpoints do not return full JSON response.

```console
$ curl -u admin:password http://localhost:8080/admin
{"_rev":"0000000054c141ca"
$ curl -u admin:password http://localhost:8080/api
{"_rev":"0000000054c141ca"
```

## Steps To Reproduce ("Repro Steps")
1. Enable HTTP Connector Handler
2. Send GET requests to HTTP endpoints

## Expected Result (Behavior You Expected to See)
Full readable JSON response.

## Actual Result (Behavior You Saw)
Incomplete JSON response.

## Environment
- **Product version:** 4.0.0-RC2
- **JRE version:** openjdk 17.0.6

**Comment by pavelhoral** [comment]
I have tried to replicate stuff and ended up with completely broken installation that refuses to start due to this error:

```
[13/Apr/2023:10:09:12 +0200] category=org.opends.messages.external severity=ERROR msgID=1 msg=Trying to redefine version: 0.0 exception=IllegalArgumentException: Trying to redefine version: 0.0 (CrestApiProducer.java:129 CrestApiProducer.java:45 AbstractRouter.java:369 AbstractRouter.java:342 AbstractRouter.java:290 Resource.java:687 SubResource.java:221 SubResource.java:141 AbstractRouter.java:361 AbstractRouter.java:342 AbstractRouter.java:290 DescribableRequestHandler.java:109 DescribableRequestHandler.java:40 AbstractRouter.java:361 AbstractRouter.java:342 AbstractRouter.java:290 FilterChain.java:273 FilterChain.java:38 InternalConnection.java:118 InternalConnection.java:26 HttpAdapter.java:849 ...)
```

Seems like there are more things broken there.

**Comment by pavelhoral** [comment]
There are multiple issues at play. However the original issue is that for the root DSE result object is resolved as `null` by [`ObjectPropertyMapper`](https://github.com/WrenSecurity/wrends/blob/9f7881593a08bbfb719dbce001be08f752dd6b76/opendj-rest2ldap/src/main/java/org/forgerock/opendj/rest2ldap/ObjectPropertyMapper.java#L321).

I am not sure if such behavior is valid - i.e. returning `null` response content with proper identifier. This leads to NPE when actually serializing the response at [`RequestRunner`](https://github.com/WrenSecurity/wrensec-commons/blob/a7d1d54ca7f761a0dfa5211b0bceecc1d3f4aa78/rest/json-resource-http/src/main/java/org/forgerock/json/resource/http/RequestRunner.java#L522).

Seems like the issue is with `RequestRunner` at a first glance.

**Comment by pavelhoral** [comment]
> I have tried to replicate stuff and ended up with completely broken installation that refuses to start due to this error:
> 
> ```
> [13/Apr/2023:10:09:12 +0200] category=org.opends.messages.external severity=ERROR msgID=1 msg=Trying to redefine version: 0.0 exception=IllegalArgumentException: Trying to redefine version: 0.0 (CrestApiProducer.java:129 CrestApiProducer.java:45 AbstractRouter.java:369 AbstractRouter.java:342 AbstractRouter.java:290 Resource.java:687 SubResource.java:221 SubResource.java:141 AbstractRouter.java:361 AbstractRouter.java:342 AbstractRouter.java:290 DescribableRequestHandler.java:109 DescribableRequestHandler.java:40 AbstractRouter.java:361 AbstractRouter.java:342 AbstractRouter.java:290 FilterChain.java:273 FilterChain.java:38 InternalConnection.java:118 InternalConnection.java:26 HttpAdapter.java:849 ...)
> ```


This has been introduced in #7 by producing more than one descriptor for the same path. Will have to go through that PR and decide what to do with that change.

**Comment by pavelhoral** [comment]
I have spent quite a lot of time on this and came to the following conclusion:

I understand the intention behind the original change performed in #7 - i.e. having read-only resources automatically produce API descriptors. However the overall change is simply not compatible with the `resource <-> path + service` mapping. I don't see any simple way to fix this without making some compromise with unwanted side effects.

I am going to create PR that will revert that change so that it does not block proper release.

**Comment by pavelhoral** [comment]
Changes merged in #79 .
